experimental/components/MsGraph/Microsoft.BotFramework.CustomAction.MsGraph.csproj

```diff
@@ -5,9 +5,22 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <!-- Nuget packaging information -->
+  <PropertyGroup>
+    <PackageId>Microsoft.Bot.Components.MsGraph</PackageId>
+    <PackageVersion>1.0.0-alpha.1</PackageVersion>
+    <Description>Plugin component for MSGraph custom action</Description>
+    <Summary>Plugin component for MSGraph custom action.</Summary>
+    <Company>Microsoft</Company>
+    <Authors>Microsoft</Authors>
+    <Product>Microsoft Bot Builder SDK</Product>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+  </PropertyGroup>
+
   <ItemGroup>
     <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.11.1" />
```

Comment:
I don't think we need to change anything here just now, but we *may* need to, and this will be a good litmus test to be sure.

Since Microsoft.Bot.Builder.Runtime pulls in a number of these dependencies, we ideally want to make sure we are leveraging those same ones. This maaaay lead to multiple assembly versions in play, so if it does, we'll need to think about that.

Again, no change here for now, but just putting that out there.